tools/ClassGen/NodeGen.cpp

```diff
@@ -318,6 +318,21 @@ int main(int argc, char **argv) {
                     "input vector of length N the output is a Nx2 matrix with "
                     "(offset, lengths) packaged for each segment.");
 
+  // clang-format off
```

Comment:
I don't think this is necessary here -- can you verify the formatting is OK without disabling `clang-format`?